ros/lib/constants.py

```diff
@@ -32,3 +27,69 @@ class CloudProvider(Enum):
 class OperatingSystem(Enum):
     """Supported Operating Systems"""
     RHEL = 'RHEL'
+
+
+class SystemStates(Enum):
```

Comment:
I can see that values ("Oversized",  "Undersized" etc) of different keys in ``SYSTEM_STATES`` is similar to elements in ``SYSTEM_STATES_EXCEPT_EMPTY``. We can avoid code redundancy by using  ``SystemStatesWithKeys.SYSTEM_STATES.value.values()`` where we are using ``SystemStates.SYSTEM_STATES_EXCEPT_EMPTY.value`` to iterate over values.